src/mlpack/methods/ann/layer/mean_pooling.hpp

```diff
@@ -160,12 +160,23 @@ class MeanPooling
   template<typename eT>
   void Pooling(const arma::Mat<eT>& input, arma::Mat<eT>& output)
   {
+    arma::mat inputPre = input;
+    size_t inRow = input.n_rows;
+    size_t inCol = input.n_cols;
```

Comment:
Looks like both parameters are only used once, so we could just use them directly.